### PR TITLE
Work on procedures, safety, and subroutines

### PR DESCRIPTION
**Pull request by gcolvin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by gcolvin** [comment]
@MicahZoltu.  The Travis CI - Pull Request is failing, apparently due to "eip-4573.md: trailing whitespace" near the end of the log.  There is no trailing white space at the end of the file, other than the single blank line that some editors insist on.  Maybe the bot is broken, maybe I misunderstand the error, or maybe somebody with write access (why not me?) can merge it anyway.

**Comment by gcolvin** [comment]
Closing, will start fresh.

**Comment by MicahZoltu** [comment]
There is no need to close the PR and create a new one.  The issue in this case is that one of your header lines has a trailing space is all.

File: EIPS/eip-4573.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ eip: 4573
 title: Entry Points and Procedures for EVM Code Sections
 description: This proposal introduces EVM procedures as entry points to EOF code sections.
 status: Draft
-type: Standards Track
+type: Standards Track
```
Comment (MicahZoltu) [inline EIPS/eip-4573.md:6]:
```suggestion
type: Standards Track
```
this should satisfy the bot.

**Pull request closed without merging**